### PR TITLE
Add config option to delete workspaces after the deployment is done

### PR DESCRIPTION
**Pull request by baixinsui**
fixes https://github.com/eclipse-xpanse/xpanse/issues/1318

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -12,4 +12,5 @@ log.terraform.stdout.stderr=true
 terraform.binary.location=
 terraform.log.level=INFO
 otel.exporter.otlp.enabled=false
-terraform.root.module.directory=
\ No newline at end of file
+terraform.root.module.directory=
+deployment.clean.workspace.enabled=true
```
Comment (swaroopar) [inline src/main/resources/application.properties:16]: we can call the property -` clean.workspace.after.deployment`.

Please also add this to the properties table in the readme file.

**Review by swaroopar — CHANGES_REQUESTED** [review]

File: src/main/resources/application.properties (hunk 1 above)
Comment (baixinsui) [reply src/main/resources/application.properties]: Updated.

File: README.md [hunk 2]
```diff
@@ -73,18 +73,20 @@ The below property names can be changed in the following ways
...
+| authorization.api.client.secret          | AUTHORIZATION_API_CLIENT_SECRET    |                                                  | The secret value of the authorization server API client                                                               |
+| authorization.swagger.ui.client.id       | AUTHORIZATION_SWAGGER_UI_CLIENT_ID |                                                  | The ID value of the authorization server swagger-ui client                                                            |
+| otel.exporter.otlp.endpoint              | OTEL_EXPORTER_OTLP_ENDPOINT        | http://localhost:4317                            | URL of the OTEL collector                                                                                             |
+| clean.workspace.after.deployment.enabled | OTEL_EXPORTER_OTLP_ENDPOINT        | true                                             | Whether to clean up the workspace after deployment is done,allowed values: true or false. Default value is true       |
```
Comment (swaroopar) [inline README.md:88]: @baixinsui the value in second column must be updated accordingly.

**Review by swaroopar — CHANGES_REQUESTED** [review]
Update var Name

File: README.md (hunk 2 above)
Comment (baixinsui) [reply README.md]: Updated.

**Review by swaroopar — APPROVED** [review]
LGTM

**Pull request merged**